src/main/resources/application.yaml

```diff
@@ -0,0 +1,30 @@
+server:
+  port: 9000
+  contextPath:
```

Comment:
Property 'server.context-path' is Deprecated: Use 'server.servlet.context-path' instead.